app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -336,16 +358,24 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     }
   };
 
-  $scope.metricSelectionChanged = function() {
-    if ($scope.emsCommonModel.metrics_selection == "disabled") {
+  $scope.tabSelectionChanged = function(tabSelector, selection) {
```

Comment:
Very nice @yaacov ! :+1: :cat2:

Reply:
Are you seating at a beach drinking from a coconut, and commenting on hard working peoples PR's ?

Reply:
nope, just your'e PR's! :wink: